app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -57,7 +57,7 @@ def query_scope
                   .includes(:cohort, :npq_course, :profile, participant_identity: [:user])
                   .where(cohort: with_cohorts)
         scope = scope.where("updated_at > ?", updated_since) if updated_since.present?
-        params[:sort].blank? ? scope.order("npq_applications.created_at ASC") : scope
```

Comment:
sorry that wasn't clear in the ticket, could we keep `scope.order("npq_applications.created_at ASC")` but remove the `params[:sort].blank?` as that param only applies to v3?